libsigner/src/messages.rs

```diff
@@ -233,6 +394,46 @@ impl StacksMessageCodec for SignerMessage {
                 let transactions = read_next::<Vec<StacksTransaction>, _>(fd)?;
                 SignerMessage::Transactions(transactions)
             }
+            SignerMessageTypePrefix::DkgResults => {
+                let aggregate_key = Self::deserialize_point(fd)?;
+                let party_polynomial_len = u32::consensus_deserialize(fd)?;
+                let mut party_polynomials = Vec::with_capacity(
+                    party_polynomial_len
+                        .try_into()
+                        .expect("FATAL: u32 could not fit in usize"),
+                );
+                for _ in 0..party_polynomial_len {
```

Comment:
Could you implement StacksMessageCodecExtensions for the party polynomial?

Reply:
Yep, added in [c50a930](https://github.com/stacks-network/stacks-core/pull/4481/commits/c50a93088d7c0261f1dbe31ab24b95028a038447)